src/world/Map/MapManagementGlobals.hpp

```diff
@@ -8,6 +8,10 @@ This file is released under the MIT license. See README-MIT for more information
 
 #include "Map/Area/AreaManagementGlobals.hpp"
 
+//////////////////////////////////////////////////////////////////////////////////////////
+// Wow build version.
+#define BUILD_VERSION 12340
```

Comment:
Can we use a const instead of define? Not 100% sure on the conventions for this.